addon/components/docs-link/index.hbs

```diff
@@ -0,0 +1,27 @@
+{{#let
+  (if
+    this.isRouteOnly
+    (component "link-to" route=@route)
+    (if
+      this.isRouteAndModelOnly
+      (component "link-to" route=@route model=@model)
+      (if
+        this.isRouteAndModelsOnly
+        (component "link-to" route=@route models=@models)
+        (component "link-to")
+      )
+    )
+  )
+  as |DocsLinkTo|
+}}
+  <DocsLinkTo
+    class="docs-md__a"
+    @query={{or @query (hash)}}
```

Comment:
Passing in an empty hash seems to trick `<LinkTo>` to behave as if nothing was passed in.
Passing in undefined triggers an error.